pkg/ui/package.json

```diff
@@ -32,5 +32,9 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11",
     "webpack-dev-server": "^3.10.3"
+  },
+  "dependencies": {
+    "css-loader": "^3.5.3",
+    "style-loader": "^1.2.0"
```

Comment:
These should technically be `devDependencies` since this app doesn't really have runtime dependencies. I'll fix this up when I add your name to the `CONTRIBUTORS.md` file.